docs/user/plagiarism-check.rst

```diff
@@ -69,18 +70,18 @@ Results
 ^^^^^^^
 After the plagiarism check was executed, the results can be inspected in different views.
 
-1. Overview of the similarity distribution. This statistical overview shows the similarity distribution based on the percentage as a histogram chart. The user can analyze the distribution quickly and adjust the plagiarism check settings as needed accordingly.
+1. Overview of the similarity distribution. This statistical overview shows the similarity distribution based on the percentage as a histogram chart. Above the chart there are additional metrics showing various statistics about the latest check. The user can analyze the distribution quickly and adjust the plagiarism check settings as needed accordingly.
```

Comment:
I would rephrase as: "Above the chart, additional metrics show various statistics about the latest check. The user can analyze the distribution quickly and adjust the plagiarism check settings accordingly."